core/domain/suggestion_registry.py

```diff
@@ -471,6 +505,45 @@ def populate_old_value_of_change(self):
         pass
 
 
+    def pre_update_validate(self, change):
+        """Performs the pre update validation. This functions need to be called
+        before updating the suggestion.
+
+        Args:
+            change: QuestionChange. The new change.
+
+        Raises:
+            ValidationError: Invalid new change.
+        """
+        if self.change.cmd != change.cmd:
+            raise utils.ValidationError(
+                'The new change cmd must be equal to %s' %
+                self.change.cmd)
+        if self.change.cmd == question_domain.CMD_UPDATE_QUESTION_PROPERTY:
```

Comment:
I think you got the CMD wrong. It should be CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION right?

Reply:
I think There are 2 types of cmd's for QuestionChange.
I have added bothe conditions here.
Could you check at line [535](https://github.com/oppia/oppia/blob/72b995615e7148576696128521423bc4f8af326f/core/domain/suggestion_registry.py#L535)

Reply:
Oh I misunderstood this. You need to ensure it is CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION. This type of suggestion corresponds to that particular command. See [here]( https://github.com/oppia/oppia/blob/72b995615e7148576696128521423bc4f8af326f/core/domain/suggestion_registry.py#L434)

Reply:
Specifically, this class corresponds to the creation of a new question, and not to update a question property. So the latter shouldn't be valid.